backend/src/test/java/com/engineering/thesis/backend/controllerIntegrationTests/PatientEPTests.java

```diff
@@ -0,0 +1,127 @@
+package com.engineering.thesis.backend.controllerIntegrationTests;
+
+//We ned to check later why we can't use PriceController directly (only wild card works).
```

Comment:
dodaj "TODO: " po //. Będzie przypominać nam w karcie TODO, że jest to do zrobienia - nie chcemy w kodzie komentarzy. Ewentualnie dodać jako issue do zrobienia, a komentarz usunąć.

Reply:
Resolved -> poszło jako TODO